monkestation/code/modules/clothing/spacesuits/hardsuits/heavy.dm

```diff
@@ -0,0 +1,82 @@
+/obj/item/clothing/head/helmet/space/hardsuit/juggernaut
+	name = "cybersun juggernaut helmet"
+	desc = "A hardened helmet plated in synthetic xenofauna hide. A biometric scanning line flashes across the visor."
+	icon_state = "hardsuit0-juggernaut"
+	hardsuit_type = "juggernaut"
+	resistance_flags = ACID_PROOF | FIRE_PROOF
+	max_heat_protection_temperature = FIRE_SUIT_MAX_TEMP_PROTECT
+	armor_type = /datum/armor/hardsuit/juggernaut
+	clothing_flags = STOPSPRESSUREDAMAGE | THICKMATERIAL | SNUG_FIT | HEADINTERNALS
+	actions_types = null
+
+/obj/item/clothing/head/helmet/space/hardsuit/juggernaut/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/radiation_protected_clothing)
+
+/obj/item/clothing/head/helmet/space/hardsuit/juggernaut/equipped(mob/living/carbon/human/user, slot)
+	..()
+	if (slot == ITEM_SLOT_HEAD)
+		var/datum/atom_hud/DHUD = GLOB.huds[DATA_HUD_DIAGNOSTIC_BASIC]
+		var/datum/atom_hud/SHUD = GLOB.huds[DATA_HUD_SECURITY_BASIC]
+		var/datum/atom_hud/MHUD = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
+		DHUD.show_to(user)
+		SHUD.show_to(user)
+		MHUD.show_to(user)
```

Comment:
do you need to call update sight here?

Reply:
that has to stay i think. otherwise whenever you take off the helmet while wearing something like health scanner huds it will remove that hud too. so the update is to restore that